src/example-types/esTwoLevelAggregation.js

```diff
@@ -30,19 +36,17 @@ module.exports = {
               context.key_data
             )
           ),
-          aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
-                _.extend(
-                  {
-                    field: context.value_field,
-                  },
+          aggs: F.arrayToObject(
+            _.identity,
+            metric =>
+              _.omitBy(_.isNil, {
+                [metric]: _.extend(
+                  { field: context.value_field },
```

Comment:
since we're modernizing this anyway, we should just convert this to use object spread

Reply:
Ok